sdk/src/Services/S3/Custom/Transfer/TransferUtilityDownloadDirectoryResponse.cs

```diff
@@ -23,8 +25,24 @@ namespace Amazon.S3.Transfer
     public class TransferUtilityDownloadDirectoryResponse
     {
         /// <summary>
-        /// The number of objects that have been downloaded
+        /// The number of objects that have been successfully downloaded.
         /// </summary>
         public long ObjectsDownloaded { get; set; }
+
+        /// <summary>
+        /// The number of objects that failed to download. Zero if all succeeded.
+        /// </summary>
+        public long ObjectsFailed { get; set; }
+
+        /// <summary>
+        /// The collection of exceptions encountered when downloading individual objects.
+        /// Only populated when the <see cref="FailurePolicy"/> is <see cref="FailurePolicy.ContinueOnFailure"/>.
+        /// </summary>
+        public IList<Exception> Errors { get; set; }
```

Comment:
why dont we populate this when its abort on failure (Even if it will always be 1 error), just to be consistent?